thetis/options.py

```diff
@@ -245,6 +245,12 @@ def __init__(self):
 
         Bottom stress is :math:`\tau_b/\rho_0 = -g \mu^2 |\mathbf{u}|\mathbf{u}/H^{1/3}`
         """
+        self.wd_alpha = None
+        r"""Coefficient or None: Wetting-drying parameter :math:`\alpha`
+
+        Used in bathymetry displacement function that ensures positive water depths. Unit is meters.
+        Default is None, which disables wetting and drying.
```

Comment:
FWIW, I'm not a fan of this style of "two different meanings" option.  I would prefer to have:

```python
self.wetting_and_drying = False
r"""Should the shallow water solver handle dry cells?

Uses the wetting and drying scheme from XXX.
If ``True``, one should also set :attr:`wd_alpha` to control the bathymetry displacement."""
self.wd_alpha = 0.1 # some reasonable default value?
r"""Coefficient: Wetting and drying parameter :math:`alpha`.

...
"""